modmesh/core.py

```diff
@@ -33,6 +33,9 @@
 
 # Use flake8 http://flake8.pycqa.org/en/latest/user/error-codes.html
 
+import os
+
+from .pylibmgr import search_library_root
```

Comment:
Use `from . import pylibmgr` and `pylibmgr.search_library_root()` later.  It will be easier to find module use outside this file during maintenance.